app/twig.php

```diff
@@ -0,0 +1,9 @@
+<?php
+require 'init.php';
+
+$pageVariables = [
```

Comment:
For each page, you should perform all your logic in the PHP file, and any variables that will be needed in the _view_ (template) should be passed in an array (`$pageVariables` in this case).